py3status/modules/speedtest.py

```diff
@@ -191,7 +191,7 @@ def _is_running(self):
         try:
             self.py3.command_output(["pgrep", "speedtest-cli"])
             return True
-        except:
+        except self.py3.CommandError:
```

Comment:
:+1: You can try `pre-push` hook now. See `.git-hooks/pre-push`.

Reply:
Activated